addons/mail/static/src/owl/components/message/message.scss

```diff
@@ -147,6 +147,19 @@
     }
 }
 
+.o_Message_trackingValues {
+    margin-top: map-get($spacers, 3);
+}
+
+.o_Message_trackingValues_value {
+    display: flex;
+    align-items: center;
+}
+
+.o_Message_trackingValues_value_item {
+    margin-inline-end: map-get($spacers, 1);
+}
```

Comment:
naming convention: `.o_Message_trackingValue` and `.o_Message_trackingValueItem`

Reply:
and done